Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```diff
@@ -145,28 +150,28 @@ configuration:
   display: Key File as Text
   name: key
   type: 4
-  hidden: true
   required: false
+  hidden: true
 - additionalinfo: Set the number of stix object that will be requested with each taxii poll (http request). A single fetch is made of several taxii polls. Changing this setting can help speed up fetches, or fix issues on slower networks. Please note server restrictions may apply, overriding and limiting the "requested limit".
   defaultvalue: '100'
   display: Max STIX Objects Per Poll
   name: limit_per_request
   type: 0
   required: false
-- additionalinfo: Choose how to handle complex observations. Two or more Observation Expressions MAY be combined using a complex observation operator such as "AND", "OR". e.g. `[ IP = 'b' ] AND [ URL = 'd' ]`
-  defaultvalue: Skip indicators with more than a single observation
+- additionalinfo: "Choose how to handle complex observations. Two or more Observation Expressions MAY be combined using a complex observation operator such as \"AND\", \"OR\". e.g. `[ IP = 'b' ] AND [ URL = 'd' ]`"
   display: Complex Observation Mode
   name: observation_operator_mode
+  type: 15
+  required: false
+  defaultvalue: Skip indicators with more than a single observation
   options:
   - Create indicator for each observation
   - Skip indicators with more than a single observation
-  type: 15
-  required: false
-- additionalinfo: "Choose whether to import the XSOAR custom fields. Note: this might overwrite the data pulled from other sources."
-  display: Update custom fields
+- display: Update custom fields
   name: update_custom_fields
   type: 8
   required: false
+  additionalinfo: "Choose whether to import the XSOAR custom fields. Note: this might overwrite the data pulled from other sources."
```

Comment:
Recommend:
"Choose whether to import Cortex XSOAR custom fields. Note: this may overwrite data pulled from other sources."